engine/swarm/key-concepts.md

```diff
@@ -7,37 +7,40 @@ title: Swarm mode key concepts
 This topic introduces some of the concepts unique to the cluster management and
 orchestration features of Docker Engine 1.12.
 
-## Swarm
+## What is a swarm?
 
 The cluster management and orchestration features embedded in the Docker Engine
-are built using **SwarmKit**. Engines participating in a cluster are
-running in **swarm mode**. You enable swarm mode for the Engine by either
+are built using **SwarmKit**. Docker engines participating in a cluster are
+running in **swarm mode**. You enable swarm mode for an engine by either
 initializing a swarm or joining an existing swarm.
 
-A **swarm** is a cluster of Docker Engines where you deploy
-[services](key-concepts.md#Services-and-tasks). The Docker Engine CLI includes the commands for
-swarm management, such as adding and removing nodes. The CLI also includes the
-commands you need to deploy services to the swarm and manage service
-orchestration.
+A **swarm** is a cluster of Docker engines, or _nodes_, where you deploy
+[services](key-concepts.md#Services-and-tasks). The Docker Engine CLI and API
+include commands to manage swarm nodes (e.g., add or remove nodes), and deploy
+and orchestrate services accross the swarm.
 
-When you run Docker Engine outside of swarm mode, you execute container
-commands. When you run the Engine in swarm mode, you orchestrate services.
+When you run Docker without using swarm mode, you execute container
```

Comment:
I'm still not totally happy with this, since you can run a non-swarm container within an Engine that is also a swarm node. Maybe just add a sentence like "You can run swarm services and standalone containers on the same Docker instance."